server/middleware/authentication.js

```diff
@@ -152,13 +153,10 @@ const _authenticateUserByJwt = async (req, res, next) => {
   }
 
   const { earlyAccess = {} } = user.collective.settings || {};
-  if (earlyAccess.dashboard) {
-    res.cookie('rootRedirect', 'dashboard', {
-      secure: true,
-      httpOnly: true,
-      sameSite: 'strict',
-      maxAge: 24 * 60 * 60 * 1000 * 365,
-    });
+  if (earlyAccess.dashboard || (parseToBoolean(config.dashboardIsDefault) && earlyAccess.dashboard !== false)) {
```

Comment:
Sorry if this was done this way based on my input but we should avoid putting config at the root level, this looks a bit unorganized. Suggesting something like `config.features.dashboard` or `config.features.dashboard.redirect`.

Reply:
Changed to `config.features.dashboard.redirect` 👍